src/ripple/app/main/Application.cpp

```diff
@@ -257,6 +258,7 @@ class ApplicationImp
     NodeStoreScheduler m_nodeStoreScheduler;
     std::unique_ptr <SHAMapStore> m_shaMapStore;
     std::unique_ptr <NodeStore::Database> m_nodeStore;
+    PendingSaves pendingSaves_;
```

Comment:
`pendingSaves_` is clearly better than `m_pendingSaves`. Are you suggesting we use an inferior style for new code just because we aren't doing a bulk search and replace?